qiime2/plugins.py

```diff
@@ -98,5 +98,11 @@ def exec_module(self, module):
                 for key, value in actions.items():
                     setattr(module, key, value)
 
+    def inspect_formats(filter=None, semantic_type=None):
```

Comment:
This utility function should be moved into the "root" of the module, so that Artifact API users can import like this:

```python
from qiime2.plugins import inspect_formats
```

Reply:
Second note, once this function is moved, you will need to add a corresponding entry to the `__all__` list of public objects (up around line 12).

Reply:
Users will need some way of specifying the enum values for the `filter` flag --- they should be able to import those variables from `qiime2.plugins`, I think.